src/test/java/org/jenkinsci/plugins/ghprb/GhprbRepositoryTest.java

```diff
@@ -109,6 +109,11 @@ public void setUp() throws Exception {
         doReturn(gt).when(trigger).getGitHub();
         
         given(gt.getRepository(anyString())).willReturn(ghRepository);
+
+        // Set rate limit before starting trigger
+        GHRateLimit initRateLimit = new GHRateLimit();
```

Comment:
https://github.com/jenkinsci/ghprb-plugin/blob/ghprb-1.30.7/src/test/java/org/jenkinsci/plugins/ghprb/GhprbRepositoryTest.java#L109  Is the line that needs to be fixed.